### DISCUSSION TITLE
RevealJS: Make the Slide Title to Be Located at the Same Place on Each Slide

### DISCUSSION THREAD

**Discussion by RoyiAvital** [quarto-dev/quarto-cli#6843] [Q&A]
### Description

I use Quarto to generate slides in RevealJS.

The problem is the location of the title of the slide is different depending on the length of the content.
Is there a way to set that the title and the text box will have a constant location?

Something similar to traditional slides templates.

**Comment by mcanouil** [comment]
Could you share a small self-contained "working" (reproducible) example to work with, _i.e._, a complete Quarto document or a Git repository? Thanks.

You can share a Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
---

This is a reproducible Quarto document using `format: html`.
It is written in Markdown and contains embedded R code.
When you run the code, it will produce a plot.

```{r}
plot(cars)
```

The end.
````
`````

**Comment by RoyiAvital** [comment]
OK, It is strange, I can't reproduce it on a simpler case to share (I can't share the document I am working on).

I will try to replicate it.

But in general, titles should be placed in the same place no matter how long the content is, right?

**Reply by mcanouil** [reply]
Without concrete example of what you mean by "same place", I cannot really answer especially if you can't reproduce on your side.

**Reply by RoyiAvital** [reply]
The title:

```qmd
## Slide Title

some content
```

To appear vertically at the same place in each slide.

**Reply by mcanouil** [reply]
Definitely something on your side until you provide a reproducible example showing what your described we can use to investigate.

**Reply by RoyiAvital** [reply]
OK, So I guess I better close this for now.
